os-profiles/microvisor-rt.yaml

```diff
@@ -6,15 +6,15 @@ metadata:
   release: 0.0.0-dev
   version: 0.3.0
 spec:
-  name: Edge Microvisor Toolkit Real Time 3.0.20250601
+  name: Edge Microvisor Toolkit Real Time 3.0.20250605
   type: OS_TYPE_IMMUTABLE
   provider: OS_PROVIDER_KIND_INFRA
   architecture: x86_64
   profileName: microvisor-rt  # Name has to be identical to this file's name
-  osImageUrl: files-edge-orch/repository/microvisor/rt/edge-readonly-rt-3.0.20250601.2058.raw.gz
-  osImageVersion: 3.0.20250601.2058
-  osImageSha256: f3dc25f85887edd83005a1481e6da1153be64e5014cbbcd5354626549718d8a5
-  osPackageManifestURL: files-edge-orch/repository/microvisor/rt/edge-readonly-rt-3.0.20250601.2058_pkg_manifest.json
+  osImageUrl: files-edge-orch/repository/microvisor/rt/edge-readonly-rt-3.0.20250605.0518.raw.gz
```

Comment:
Do not see the image there yet.
![image](https://github.com/user-attachments/assets/a154664a-0ad6-4ebd-b57b-54418038b50b)

Reply:
yep, already did contacts the EMT team,